open-banking-brasil-financial-api-1_ID2-ptbr.md

```diff
@@ -296,9 +302,38 @@ Além disso, o cliente confidencial
 
-Os participantes devem apoiar todas as considerações de segurança especificadas na cláusula 8 [Financial-grade API Security Profile 1.0 - Parte 2: Advanced] [FAPI-1-Advanced] e o [Manual de Segurança de Banco Central do Brasil] (https://www.bcb.gov.br/estabilidadefinanceira/exibenormativo?tipo=Instru%C3%A7%C3%A3o%20Normativa%20BCB&numero=99). O ICP brasileiro emite certificados RSA x509 somente, portanto, para simplificar, a seção remove o suporte para algoritmos EC e exige que apenas algoritmos de criptografia recomendados pela IANA sejam usados.
+Os participantes devem apoiar todas as considerações de segurança especificadas na cláusula 8 [Financial-grade API Security Profile 1.0 - Parte 2: Advanced] [FAPI-1-Advanced] e o [Manual de Segurança de Banco Central do Brasil] (https://www.bcb.gov.br/estabilidadefinanceira/exibenormativo?tipo=Instru%C3%A7%C3%A3o%20Normativa%20BCB&numero=134). O ICP brasileiro emite certificados RSA x509 somente, portanto, para simplificar, a seção remove o suporte para algoritmos EC e exige que apenas algoritmos de criptografia recomendados pela IANA sejam usados.
+
+## Considerações sobre assinatura do conteúdo de mensagens (JWS) {#jws}
+
+1. Para garantir a integridade e o não-repúdio das informações tramitadas em **API´s sensíveis e que indicam essa necessidade na sua documentação**, deve ser adotado a estrutura no padrão JWS definida na [RFC7515] e que inclui:
+   * Cabeçalho (_JSON Object Signing and Encryption – JOSE Header_), onde se define o algoritmo utilizado e inclui informações sobre a chave pública ou certificado que podem ser utilizadas para validar a assinatura;
+   * Payload (_JWS Payload_): conteúdo propriamente dito e detalhado na especificação da API;
+   * Assinatura digital (_JWS Signature_): assinatura digital, realizada conforme parâmetros do cabeçalho.
+2. Cada elemento acima deve ser codificado utilizando o padrão Base64url [RFC4648](https://tools.ietf.org/html/rfc4648#section-5) e, feito isso, os elementos devem ser concatenados com “.” (método JWS Compact Serialization, conforme definido na [RFC7515]).
+
+3. O payload das mensagens (_Request Object_ e _Response Object_) assinadas devem incluir as seguintes `claims` presentes na [RFC7519] (JWT):
+   * **aud** (no _Request Object_): o Provedor do Recurso (p. ex. a instituição detentora da conta) deverá validar se o valor do campo **aud** coincide com o endpoint sendo acionado;
+   * **aud** (no _Response Object_): o cliente da API (p. e. instituição iniciadora) deverá validar se o valor do campo **aud** coincide com o seu próprio `organisationId` listado no diretório;
+   * **iss** (no Request Object e no Response Object): o receptor da mensagem deverá validar se o valor do campo **iss** coincide com o `organisationId` do emissor;
+   * **jti** (no Request Object e no Response Object): o valor do campo **jti** deverá ser preenchido com o UUID definido pela instituição de acordo com a RFC 4122 usando o versão 4;
```

Comment:
https://datatracker.ietf.org/doc/html/rfc7519#section-4.1.6 I suggest referring to the actual specifications for iat. as the format of the iat must follow the definition of iat.

it's not a timestamp it is a numeric from epoch